lnbits/tasks.py

```diff
@@ -149,37 +148,36 @@ async def check_pending_payments():
     incoming = True
 
     while True:
-        async with db.connect() as conn:
-            logger.info(
-                f"Task: checking all pending payments (incoming={incoming},"
-                f" outgoing={outgoing}) of last 15 days"
-            )
-            start_time = time.time()
-            pending_payments = await get_payments(
-                since=(int(time.time()) - 60 * 60 * 24 * 15),  # 15 days ago
-                complete=False,
-                pending=True,
-                outgoing=outgoing,
-                incoming=incoming,
-                exclude_uncheckable=True,
-                conn=conn,
-            )
-            for payment in pending_payments:
-                await payment.check_status(conn=conn)
+        logger.info(
+            f"Task: checking all pending payments (incoming={incoming},"
+            f" outgoing={outgoing}) of last 15 days"
+        )
+        start_time = time.time()
+        pending_payments = await get_payments(
+            since=(int(time.time()) - 60 * 60 * 24 * 15),  # 15 days ago
```

Comment:
- consider optimizing the `since` value
```py
since=(int(time.time()) - 60 * 60 * 24 * 15), # 15 days ago
while True:
    # ...
        last_check_time = int(time.time())
        pending_payments = await get_payments(
            since=since,
            # ...
        )
        since = last_check_time
        # ...
     # ...
```

Reply:
The 15 days are meant to cover the maximum expiry time for bolt11 invoices. Invoices checked once can be paid when checked another time so we can't change this.